markdownlint/docs/TOP010.md

```diff
@@ -0,0 +1,29 @@
+# `TOP010` - Use lazy numbering within ordered lists
+
+Tags: `ordered-list, lazy-numbering`
+
+Aliases: `use-lazy-numbering-in-ordered-lists`
+
+This rule is triggered when lazy-numbering is not used in ordered lists.
```

Comment:
Think we could probably elaborate *a little* more on what "lazy-numbering" is, e.g. we're specifically wanting only `1.` prefixes for ordered lists and why (convenience when changing ordered lists since the markdown converter is smart and will set the right numbers automatically).